src/cuga/backend/slash_commands/dispatcher.py

```
@@ -0,0 +1,297 @@
+            name="slash_command",
+            as_type="span",
+            input={"raw_input": parsed.raw_input, "args": parsed.raw_args},
+            output={
+                "resolved_kind": result.kind,
+                "resolved_name": result.resolved_name,
+                "top_suggestions": top_suggestions,
+            },
+            metadata={"duration_ms": duration_ms},
+        )
+        span.end()
+    except Exception:  # pragma: no cover - telemetry is best-effort
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)